ckan/lib/helpers.py

```diff
@@ -896,7 +897,7 @@ def link_to(label, url, **attrs):
 
 
 @core_helper
-def file(name, value=None, id=None, **attrs):
+def file(name, value='', id=None, **attrs):
```

Comment:
choose a different name so we don't shadow the builtin `file` object

Reply:
I think, that `file` was completely removed in Python 3 in favor of `open`. And this helper doesn't seem to be very popular - I don't see it in the core, neither is saw it inside any of extensions. So i prefer to remove it completely instead of renaming

Reply:
even better